test/Microsoft.ServiceFabric.Actors.IntegrationTests/XUnitSettings.cs

```diff
@@ -0,0 +1,6 @@
+// ------------------------------------------------------------
```

Comment:
Rename all XUnitSettings.cs files to AssemblyInfo.cs. 
If a project already contains an AssemblyInfo.cs, merge the contents.
Add using Xunit directive and shorten the attribute name.